R/TealAppDriver.R

```diff
@@ -73,7 +73,7 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
     #' @param ... arguments passed to parent [`shinytest2::AppDriver`] `click()` method.
     click = function(...) {
       super$click(...)
-      self$wait_for_idle(5000)
+      self$wait_for_idle()
```

Comment:
@averissimo who set 5000 in here :D?

Reply:
Oh I should've run git blame first. Do you think that we need to change the duration parameter from 500ms to something else?

Or was this a confusion with the `timeout` parameter?

Reply:
Note that the `duration` is the milliseconds duration that the UI needs to be stable to be considered idle.

It will try to do this for 20s (as that's the default timeout value for `TealAppDriver`)

Reply:
That was me. I thought we wanted to increase the wait time for idle.
https://github.com/insightsengineering/teal/issues/1195#issuecomment-2047788486

Reply:
Currently, the timeout is being set to 20s, so I think we can merge this and change it appropriately if the integration fails.